plugins/postcss-minify/src/index.ts

```diff
@@ -75,86 +85,107 @@ function removeEmptyNodes(node: Container | Document): boolean {
 	return false;
 }
 
+function setSemicolon(node: AtRule | Rule) {
+	if (node.raws.semicolon) {
+		const last = node.last;
+		if (last?.type !== 'decl' || !last.variable) {
+			node.raws.semicolon = false;
+		}
```

Comment:
Technically the last semicolon is always safe to omit, even for variables.
But out of an abundance of caution I am skipping them.

Reply:
Curious what made you go this way. I'll always agree with extra caution just curious about what could be the factor!